supervision/detection/line_counter.py

```diff
@@ -13,19 +14,23 @@ class LineZone:
     Count the number of objects that cross a line.
     """
 
-    def __init__(self, start: Point, end: Point):
+    def __init__(self, start: Point, end: Point, direction="both"):
```

Comment:
I love your added `direction` argument. I don't like `direction` as a string. We can solve that by:

1. Using enum

    ```python
    class Direction(Enum):
        IN = "in"
        OUT = "out"
        BOTH = "both"
    ```
    
    than:
    
    ```python
    def __init__(self, start: Point, end: Point, direction: Direction):
    ```

2. Using two separate bool arguments (preferred):

    ```python
    def __init__(
        self, 
        start: Point, 
        end: Point, 
        trigger_in: bool = True, 
        trigger_out: bool = True
    ):
    ```

Please update the docstring as well to reflect that change.